.devcontainer/devcontainer.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Wing Development",
+	"image": "mcr.microsoft.com/devcontainers/base:jammy",
+  "postStartCommand": "volta install node npm && volta setup && npm install",
```

`volta install node npm` does not install the pinned versions. Should be sufficient to just install

```suggestion
  "postStartCommand": "volta setup && npm install",
```

---

Ah yupp I see my confusion, 👍 updated